ajax/libs/react-toolbox/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-toolbox",
+  "description": "A set of React components implementing Google's Material Design specification with the power of CSS Modules.",
+  "homepage": "http://www.react-toolbox.com",
+  "version": "1.3.4",
+  "author": {
+    "name": "React Toolbox Team",
+    "url": "http://github.com/react-toolbox"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/react-toolbox/react-toolbox.git"
+  },
+  "keywords": [
+    "components",
+    "material design",
+    "react",
+    "react-component",
+    "toolkit"
+  ],
+  "license": "MIT",
+  "filename": "index.min.js",
+  "npmName": "react-toolbox",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```

Comment:
@sufuf3 I think we can use `"{**/*,*}.!(scss|*ts)"`. You should try to fetch files by using auto-updater, otherwise you will have wrong config sometimes.